bindings/python/iota_sdk/types/token_scheme.py

```diff
@@ -17,9 +18,15 @@ class SimpleTokenScheme:
         maximum_supply: The maximum supply of the token.
         type: The type code of the token scheme.
     """
-    minted_tokens: HexStr
-    melted_tokens: HexStr
-    maximum_supply: HexStr
+    minted_tokens: int = field(metadata=config(
+        encoder=str
```

Comment:
Does that actually encode to prefixed hex ?

Reply:
oh, no, I didn't realize it was hex

Reply:
decoding from hex works by default?

Reply:
I don't think so? but tbh I have no idea

Reply:
it did not, but I fixed it